opencti-platform/opencti-front/src/utils/Time.js

```diff
@@ -82,6 +82,12 @@ export const dateFormat = (data, specificFormat = null) => {
     : '';
 };
 
+export const validStartTimeForRelationCreation = (time) => {
+  const date = new Date(time);
+  date.setMinutes(date.getMinutes() - 1);
```

Comment:
In the issue Nino ask for one second less instead of one minute, was there any changes?

Reply:
because second was not taken into account

Reply:
I checked with @nino-filigran . 1 minute is ok.
we don't display second tht is why i used minute.